### PR TITLE
Fix contextual boundary

### PR DESCRIPTION
**Pull request by andybak**
I misread the docs originally #739 was a mistake (it turned the boundary off all the time.

Draft PR and currently untested. I'll wait for the build to finish and then try it.

### CODE REVIEW

File: Assets/Editor/BuildTiltBrushPostProcess.cs [hunk 1]
```diff
@@ -69,13 +69,13 @@ public void OnPostGenerateGradleAndroidProject(string path)
...
-#if ENABLE_BOUNDARYLESS_APP
-            UnityEngine.Debug.Log("Add com.oculus.feature.BOUNDARYLESS_APP");
+#if ENABLE_CONTEXTUAL_BOUNDARYLESS_APP
+            UnityEngine.Debug.Log("Add com.oculus.feature.CONTEXTUAL_BOUNDARYLESS_APP");
             AddOrRemoveTag(doc,
                     androidNamespaceURI,
                     "/manifest",
                     "uses-feature",
-                    "com.oculus.feature.BOUNDARYLESS_APP",
+                    "com.oculus.feature.CONTEXTUAL_BOUNDARYLESS_APP",
```
Comment (mikeskydev) [inline Assets/Editor/BuildTiltBrushPostProcess.cs:78]: I think this feature/permission may be added automatically if the ovrsettings file is set up correctly: https://developer.oculus.com/documentation/unity/unity-boundaryless/#contextual-boundaryless-implementation 
image

Comment (andybak) [reply Assets/Editor/BuildTiltBrushPostProcess.cs:78]: It seems to work as it is so is this just a cleanup?

**Comment by andybak** [comment]
Tested and working

**Comment by mikeage** [comment]
Failed on Q2 upload :-(

See PR #735 and #737

**Comment by andybak** [comment]
There was obviously a conversation or thought process between those two commits.

And I obviously felt it was either going to be so memorable or so easy to find that it wasn't worth documenting it in the PR discussion itself.

Sigh.

**Pull request merged**